package.json

```diff
@@ -16,10 +16,19 @@
     "smp": "./bin/smp.js"
   },
   "type": "module",
+  "files": [
+    "bin",
+    "dist",
+    "lib",
+    "map-viewer"
+  ],
```

Comment:
not sure about including `map-viewer`. anything else to include here otherwise?

Reply:
`map-viewer` makes sense to include since one of the bin scripts uses it. still an open question about anything else that should be added to this list...

Reply:
think that should be it